src/WPF/WPF.Viewer/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.xaml.cs

```diff
@@ -44,10 +48,19 @@ public partial class GeodatabaseTransactions
         // Store a reference to the table to be edited.
         private GeodatabaseFeatureTable _editTable;
 
+        // Flag indicating if an edit exists which is not committed and was created during a transaction.
+        // Not to be confused with HasLocalEdits() which is for local edits not synchronized with the service.
+        private bool _hasUncommittedEdits;
```

Comment:
I don't think we need this. `Geodatabase.HasLocalEdits()` or `GeodatabaseFeatureTable.HasLocalEdits()` can be used instead.

Reply:
This handles the case where user has unsynced edits (HasLocalEdits() == true), so initially they created a feature. Then, they tap add feature button, but before they tap the map, they tap stop editing button. The prompt makes it seems like there are changes to commit locally, but there isn't anything from that editing session. Those changes were already locally committed in a previous transaction. Let me know if we want to address this differently.

![use-case](https://github.com/Esri/arcgis-maps-sdk-dotnet-samples/assets/118313312/98172739-9685-4046-b0f1-881bd48ce17f)

Reply:
I still think we can do without this property based on how it is used.
May be you want to permit `geometryEditor.Stop()` but do not want g`eodatabase.CommitTransaction()` or `RollbackTransaction()`... in those cases, former can use geometryEditor.IsStarted and latter can use `geodatabase.HasLocalEdits()`

Reply:
Per our discussion, got rid of this added conditional.